### DISCUSSION TITLE
train/water_drop/

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#948] [Announcements]
# train/water_drop/

https://ieda.oscc.cc/train/water_drop/

**Comment by Xingquan-Li** [comment]
欢迎大家报名水滴计划

**Reply by wrz1999** [reply]
你好，请问怎么才能报名水滴计划呢？

**Comment by Xingquan-Li** [comment]
今年还没开始，预计7月份才会组织，欢迎继续关注iEDA官网和微信群
Best regards,
Xingquan

——————————
Xingquan Li,
Ph.D,

Pengcheng National Lab, Shenzhen 518000,
China,
Email: ***@***.***, ***@***.***

**Reply by wrz1999** [reply]
好的 到时候报名

**Reply by wy-source** [reply]
请问这个项目，已经毕业的可以报名吗？

**Reply by Xingquan-Li** [reply]
可以的，不过今年水滴计划的时间还没定。等确定后，我们会在官网这边和我们微信群通知，请关注。
